Header header
int8 readings
float64 scale_ratio
float64 raw_data_mean
float64 data_mean
float64 weight_mean
float64 force_mean
float64 last_raw_data